metadata.tex

```diff
@@ -294,6 +294,8 @@
 
 \include{\sections/combat.tex}
 
+% Empty page placeholder
```

Comment:
```suggestion
\iftoggle{printable}{\include{....}}{}
```

Reply:
I was going to cleanup Notes after release, but since you want to add yet another empty page, there's no reason to use this way for it from the start

Reply:
Thanks for the refresher.